pkg/kubectl/cmd/exec.go

```diff
@@ -159,6 +160,11 @@ func (p *ExecOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, argsIn []s
 		}
 		p.Command = argsIn
 	} else {
+		if strings.HasPrefix(argsIn[0], "pods/") {
```

Comment:
It seems that the [proposing issue](https://github.com/kubernetes/kubernetes/issues/65670) wants the prefix to be `pod/`?

Also, please fix the release note.

Reply:
string.TrimPrefix would be simpler to follow

Reply:
+1 for `strings.TrimPrefix`